bundler.gemspec

```diff
@@ -47,7 +47,6 @@ Gem::Specification.new do |s|
   s.add_development_dependency "rubocop",    "= 0.65.0"
 
   s.files = Dir.glob("{lib,exe}/**/*", File::FNM_DOTMATCH).reject {|f| File.directory?(f) }
-
```

Comment:
This blank line was removed due to a bad conflict resolution, I'll leave it as it is.